templates/base.html

```diff
@@ -35,7 +35,8 @@
 					   	</button>
 					</form>
 					<form class="navbar-form navbar-right" action="/logout" method="post">
-						<button class="btn btn-lg btn-danger" type="submit">
```

Comment:
Clicking on my own username that is displayed in the nav-bar will end up registering the click on the logout button.

**This is not supposed to happen, please fix.**